cypress/integration/User_test.js

```diff
@@ -0,0 +1,84 @@
+
+describe('Testing homepage', () => {
+
+  beforeEach(() => {
+
+    cy.visit('http://localhost:3000/')
+
+    cy.intercept({
+      method: 'GET',
+      url: 'http://localhost:3001/users/1'
+    }, {
+      statusCode: 201,
```

Comment:
This was a tough one... Im going to look into why we needed the status code , but I am thinking it is because that is how our implementation logic was setup